ODIN_II/SRC/odin_error.cpp

```diff
@@ -94,7 +93,7 @@ void _log_message(odin_error error_type, long column, long line_number, long fil
             file_name = file_name.substr(slash_location + 1);
```

Comment:
I've done filename only here since otherwise I would need a more complex regex in the parse_conf, but if you see some value to it, I'm not against switching the format.

Reply:
My IDE (CLion) worked if you printed the file name by itself or the entire path. I cant speak for other IDEs though. So this was fine

Reply:
Then I guess it's good as is!